README.md

```diff
@@ -3,11 +3,16 @@ Web API service for Matterhorn implemented with Go.
 
 ## Setup
 
+### Database
+MatterhornApiService expects to connect to a MySQL-compatible database instance. You will need to set some environment variables to define how to connect.
+`MATTERHORN_DB_ENDPOINT` - Specifies the host name of the database instance, including the port. E.g., http://mydatabase.com:3306.
+`MATTERHORN_DB_PASSWORD` - Specifies the password used to connect to the database instance. User name is expected to be "admin" for now.
```

Comment:
Should we provide the location for the password, ie check slack or ask Matt

Reply:
Nah, let's keep track of those sorts of operational logistics independently of the code.

In practice going forward you (or anybody who clones this repo) will be able to create a compatible database instance. The fact that we currently intend to use this with a particular Amazon RDS database instance is incidental to the implementation.